opus/application/static_media/js/opus.js

```diff
@@ -135,11 +139,37 @@ var opus = {
 
         } else {
             // selections in the url hash is different from opus.last_selections
-            // reset the pages: {"gallery":1, "data":1, "cart_gallery":1, "cart_data":1 };
+            // reset the pages:
             opus.prefs.page = default_pages;
 
-            // and reset the query:
-            o_browse.resetQuery();
+            // create an object from selections to compare with opus.selections
+            let modifiedSelections = {};
+            $.each(Object.keys(selections), function(idx, slug) {
+                // qtype is not updated in opus.selections
```

Comment:
I don't understand this code or this comment. What does it mean for qtype to not "be updated"?